lib/std/zig/parse_string_literal.zig

```diff
@@ -64,6 +64,37 @@ pub fn parseStringLiteral(
                     try list.append('"');
                     state = State.Start;
                 },
```

Comment:
This seems to be missing single quote, could you add it here?

Reply:
All done in 49ac696